charts/logging-operator/values.yaml

```diff
@@ -136,7 +136,7 @@ logging:
   enabled: false
 
   # -- Reference to the logging system. Each of the loggingRefs can manage a fluentbit daemonset and a fluentd statefulset.
-  loggingRef: ""
+  loggingRef: "logging"
```

Comment:
can we leave this empty for the sake of backward compatiblity with the existing tests?